ciao-cli/instance.go

```diff
@@ -54,6 +54,7 @@ const (
 	}
 	SSHIP   string                                // Instance SSH IP address
 	SSHPort int                                   // Instance SSH Port
+	OsExtendedVolumesVolumesAttached []string     // list of attached volumes
```

Comment:
@tpepper it's named the way it is to match the openstack variable name as defined in their spec.